Source/FieldSolver/SpectralSolver/SpectralKSpace.cpp

```diff
@@ -233,37 +254,37 @@ SpectralKSpace::getModifiedKComponent( const DistributionMapping& dm,
  * `nodal` indicates whether this finite-difference approximation is
  * taken on a nodal grid or a staggered grid.
  */
-Vector<Real>
-getFonbergStencilCoefficients( const int n_order, const bool nodal )
-{
-    AMREX_ALWAYS_ASSERT_WITH_MESSAGE( n_order%2 == 0,
-                                      "n_order should be even.");
-    const int m = n_order/2;
-    Vector<Real> coefs;
-    coefs.resize( m+1 );
+ Vector<Real>
```

Comment:
Could you remove the whitespace at the beginning of each line (probably added by accident by the text editor or something) in this function body? It seems this function was not modified at all otherwise, right?